src/elements/content-uploader/ItemList.js

```diff
@@ -63,15 +64,15 @@ const ItemList = ({
                         cellRenderer={actionCell}
                         dataKey="status"
                         flexShrink={0}
-                        width={onUpgradeCTAClick ? 100 : 25}
+                        width={actionCellWidth}
                     />
                     {isResumableUploadsEnabled && (
                         <Column
                             className="bcu-item-list-action-column"
                             cellRenderer={removeCell}
                             dataKey="remove"
                             flexShrink={0}
-                            width={25}
+                            width={32}
```

Comment:
I was thinking more of storing 100/32 as a constant so that it can be used in both places